test/onnx/onnx_test.cpp

```diff
@@ -761,7 +761,9 @@ TEST_CASE(constant_empty_scalar_int64_test)
 {
     migraphx::program p;
     auto* mm = p.get_main_module();
-    mm->add_literal(migraphx::literal{migraphx::shape::int64_type});
+    migraphx::shape empty_shape{migraphx::shape::int64_type, {1}, {0}};
```

Comment:
This doesn't look like an empty shape to me. It looks like a scalar, as it has a length of 1.

Reply:
Correct the intention was to make empty shapes -> scalar of length 1 with value 0

Reply:
I think you want an empty shape to have a length of 0 if you're avoiding having `shape.lens() == {}`? With this how would I represent a scalar zero?